### ISSUE TITLE
Conflict with Rank Math SEO

### ISSUE THREAD

**Issue by lupett** [soflyy/oxygen-bugs-and-features#955]
Conflict with Rank Math SEO
When you try to reply a comment simply don't work reply feature.

**To Reproduce**
1. Install WP + Oxygen + Rank Math SEO
2. Make a new blog template with comment list and comment box
3. Comment Hello World post
4. Try to reply before comment 
5. New comment will be posted as new comment and not as indented comment

I've tried on Chrome - Edge Chrome

**Comment by KittenCodes** [comment]
@lupett This issue persists when Oxygen is disabled. Please contact Rank Math.

**Comment by lupett** [comment]
@KittenCodes so now is confirmed? :)

**Comment by KittenCodes** [comment]
@lupett Yes, it is a confirmed bug - however, I'm not yet sure if the cause is Oxygen or Rank Math.

**Comment by lupett** [comment]
@KittenCodes maybe javascript conflict

**Comment by the-stew** [comment]
@lupett Spoke to RankMath support, add this via a plugin like Code Snippets: https://rankmath.com/kb/filters-hooks-api-developer/#remove-replytocom-query

`/**
 * Filter: 'rank_math_remove_reply_to_com' - Allow or disable the Rank Math feature that removes `?replytocom` 
 * query parameters from URLs.
 * @param bool $return True to remove, false not to remove.
 */
add_filter( 'rank_math/frontend/remove_reply_to_com', '__return_false');`
